content/advanced/330_servicemesh_using_appmesh/virtual_gateway_setup/add_virtual_gateway.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Add Virtual Gateway"
+date: 2020-01-27T08:30:11-07:00
+weight: 50
+---
+
+#### Adding App Mesh Virtual Gateway
+
+![gateway](/images/app_mesh_fargate/meshify3.png)
+Until now we have verified the communication between services deployed in `Managed Nodegroup` pod and `Fargate` pod is going via envoy proxy, lets expose the frontend service `frontend-node` using App Mesh virtual gateway.
```

Comment:
can you swap "is going" here out for something like 'is proxied by' or 'is routed through'?

Reply:
updated